enhancements/arbiter-clusters.md

```diff
@@ -0,0 +1,597 @@
+6. The installer sets the control plane and infrastructure topology to
+   `HighlyAvailableArbiter`
+7. With the object `arbiterNode` in the install config, the installer creates
+   the machine object with the correct node labels and machine config resources
+   for ignition bootstrap files.
+8. The installer applies the taint to the arbiter MachineSet
+9. The user can make any alterations to the node machine type to use less
+   powerful machines.
+10. The user then begins the install via `openshift-install create cluster`
+
+#### During Install
+
+1. The CEO will watch for new masters and the arbiter role
+2. CEO will create the operand for the etcd deployments that have tolerations
+   for the arbiter
+3. Operators that have tolerations for the arbiter should be scheduled on the
+   node
+4. The install should proceed as normal
+
+### API Extensions
+
+#### Installer API Change
+
+The `installConfig` will include a `installConfig.arbiterNode` object for a
+`MachinePool` to configure arbiter infra structure.
+
+#### OCP Config API Change
+
+The Infrastructure config fields for `ControlPlaneTopology` and
+`InfrastructureTopology` will support the new value of `HighlyAvailableArbiter`
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+At the time being there is no impact on Hypershift since this edge deployment
+will require running the control plane.
+
+#### Standalone Clusters
+
+This change is relevant to standalone deployments of OpenShift at the edge or
+datacenters. This enhancement specifically deals with this type of deployment.
+
+#### Single-node Deployments or MicroShift
+
+This change does not effect Single Node or MicroShift.
+
+### Implementation Details/Notes/Constraints
+
+Currently there are some behavior unknowns, we will need to test out our POC to
+validate some of the desires in this proposal. In it's current version this
+proposal is not exhaustive but will be filled out as we implement these goals.
+
+We currently expect this feature to mainly be used by `baremetal` installs, or
+specialized hardware that is built to take advantage of this type of
+configuration. In the current design we make two paths for cloud and baremetal
+installs in the installer. However, the cloud install is primarily for testing,
+this might mean that we simplify the installer changes if we are the only ones
+using cloud installs, since we can simply alter the manifests in the pipeline
+with out needing to change the installer.
+
+In the current POC, a cluster was created on AWS with 2 masters running
+`m6a.4xlarge` and 1 arbiter running a `m6a.large` instance. At idle the node is
+consuming `1.41 GiB / 7.57 GiB	0.135 cores / 2 cores`. The cluster installs and
+operates with the following containers running on the arbiter node. This seems
+very promising, but we will need to identify any other resources we might want
+to run or remove from this list.
+
+```
+"openshift-cluster-csi-drivers/aws-ebs-csi-driver-node-qvvqc"
```

Comment:
probably not needed - no required component provisions EBS volumes

Reply:
Thanks for the info, adding it to a table with this list to help keep track.